src/services/config-service.ts

```
@@ -0,0 +1,180 @@
+    this.saveConfig(updatedConfig, projectRoot);
+    return updatedConfig;
+  }
+  
+  /**
+   * Check if index exists
+   */
+  static indexExists(projectRoot?: string): boolean {
+    const config = this.loadConfig(projectRoot);
+    return fs.existsSync(config.output);
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)